extern/wine/COPYING.LIB

```diff
@@ -0,0 +1,502 @@
+		  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
same thing here - I don't see such requirement. we don't want unnecessary bloat
If we really need to supply licenses, I'd suggest to create a separate file mentioning all the external sources we use directly with links to corresponding licenses

Reply:
Removed.